modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -1171,13 +1374,23 @@ struct InferROI: public cv::detail::KernelTag {
             if (!input_reshape_table.empty()) {
                 const_cast<IE::CNNNetwork *>(&uu.net)->reshape(input_reshape_table);
             }
+
+            // NB: configure input param for further preproc
+            if (uu.net_input_params.is_applicable(mm)) {
+                const_cast<IEUnit::InputFrameDesc &>(uu.net_input_params).set_param(input_name, ii);
```

Comment:
Does this code deserve own function which will wrap if(load){} else{import} case? Numbers of Mat\Frame inputs in ctx can bring difficulties.